modules/gapi/misc/python/package/gapi/__init__.py

```diff
@@ -299,3 +332,10 @@ def kernel_with_params(cls):
 
 
 cv.gapi.wip.GStreamerPipeline = cv.gapi_wip_gst_GStreamerPipeline
+
+class TrackingStatus:
```

Comment:
(Optional) 

What type is used for `NEW`, `LOST`, `TRACKED`? I assume `int`, will there be just `0`,`1`,`2` when we print it into console?
Perhaps we can leverage python `enum` module to improve that.